docs/040_pepr-tutorials/040_pepr-gitops-workflow.md

```diff
@@ -0,0 +1,5 @@
+# Using Pepr in a GitOps Workflow
+
+Admission Controllers and Operators can be complimentary to GitOps workflows. The important part is to ensure that the GitOps workflow is not disrupted by the admission controller or operator. This can be achieved by clearly defining boundaries in the GitOps Server in configuration.
+
+For the complete tutorial, please refer to [pepr-with-argo](https://github.com/cmwylie19/pepr-with-argo).
```

Comment:
Is there a reason we link to a tutorial in a personal repo instead of something that will be displayed on our docs site? I think that this could be an good example of a tutorial to add under `docs/040_pepr-tutorials/`

Reply:
i would rather have it in a Defense Unicorns repo to be honest but i think it is going to be harder to maintain if it is not a real repo somewhere with dependabot that gets updates and is easily runnable

Reply:
but im pretty sure we do not put POC things in Defense Unicorns repos anymore since we had like hundreds at one point and cleaned them up

Reply:
also, since it is ArgoCD it needs to point to a real repo since that is how argo works.

Reply:
@cmwylie19 is this still on hold due to the labs org?